inst/resources/activate.R

```diff
@@ -8,6 +8,21 @@ local({
   # the project directory
   project <- getwd()
 
+  # use start-up diagnostics if enabled
+  diagnostics <- Sys.getenv("RENV_STARTUP_DIAGNOSTICS", unset = "FALSE")
+  if (diagnostics) {
+    start <- Sys.time()
+    profile <- tempfile("renv-startup-", fileext = ".Rprof")
+    utils::Rprof(profile)
+    on.exit({
+      utils::Rprof(NULL)
+      elapsed <- signif(difftime(Sys.time(), start, units = "auto"), digits = 2L)
+      writeLines(sprintf("- renv took %s to run the autoloader.", format(elapsed)))
+      writeLines(sprintf("- Profile: %s", profile))
+      print(utils::summaryRprof(profile))
```

Comment:
```suggestion
      print(utils::summaryRprof(profile))
      unlink(profile)
```
?

And maybe also `the$startup_profile <- utils::summaryRprof(profile)` so that if needed we can reach inside and interactively inspect the profile?

Reply:
I wanted to keep the profile around on disk just in case it's worth sharing the raw data in some scenarios. Since it lives in the temporary directory, it'll be cleaned up when the user quits anyhow.